### PR TITLE
Schema-generator docker container

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

**Comment by fbertsch** [comment]
Related PR: https://github.com/mozilla/mozilla-schema-generator/pull/18

File: dags/probe_scraper.py [hunk 1]
```diff
@@ -13,17 +15,37 @@
     'retry_delay': timedelta(minutes=30),
 }
 
-dag_daily = DAG('probe_scraper',
+with DAG('probe_scraper',
                 default_args=default_args,
-                schedule_interval='@daily')
+                schedule_interval='@daily') as dag:
+
+    probe_scraper = EMRSparkOperator(
```
Comment (haroldwoo) [inline dags/probe_scraper.py]: Since this is new, we may want to consider using dataproc

File: dags/probe_scraper.py [hunk 2]
```diff
@@ -13,17 +15,37 @@
...
+        job_name="Probe Scraper",
+        execution_timeout=timedelta(hours=4),
+        instance_count=1,
+        email=['telemetry-client-dev@mozilla.com', 'gfritzsche@mozilla.com', 'aplacitelli@mozilla.com', 'frank@mozilla.com'],
+        env={},
+        uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/probe_scraper.sh",
+        output_visibility="public",
+        dag=dag)
+
+    schema_generator = kubernetes_pod_operator.KubernetesPodOperator(
```
Comment (haroldwoo) [inline dags/probe_scraper.py]: consider using gkepodoperator, which is derived from k8spodoperator.  We are already using it in some dags
e.g. https://github.com/mozilla/telemetry-airflow/blob/master/dags/mango_log_processing.py#L194 and Jason has overridden some methods to fix a permissions issue
https://github.com/mozilla/telemetry-airflow/blob/master/dags/operators/gcp_container_operator.py

File: dags/probe_scraper.py (hunk 1 above)
Comment (fbertsch) [reply dags/probe_scraper.py]: This isn't new, I just moved it a bit. Let's keep it in AWS until it's ready for GCP.

**Comment by fbertsch** [comment]
Successfully tested and the generated-schemas branch was updated: https://github.com/mozilla-services/mozilla-pipeline-schemas/tree/generated-schemas

**Pull request merged**